docker/Dockerfile.py

```diff
@@ -34,6 +34,12 @@
 
     RUN ln -s /home/s3am/bin/s3am /usr/local/bin/
 
+    RUN sudo apt-get -y remove docker-engine
```

Comment:
This is ugly and it means that we are using the wrong base image. This should be fixed as part of #1162.